From david-b@pacbell.net Tue Dec  5 10:09:54 2006
From: Andrew Victor <andrew@sanpeople.com>
To: linux-usb-devel@lists.sourceforge.net
Subject: USB: ohci at91 warning fix
Date: Tue, 5 Dec 2006 03:20:31 -0800
Cc: Greg KH <greg@kroah.com>,
 Andrew Victor <andrew@sanpeople.com>
Message-Id: <200612050320.32627.david-b@pacbell.net>

From: Andrew Victor <andrew@sanpeople.com>

Remove a warning about an unused variable in the OHCI bus glue for at91.

Signed-off-by: Andrew Victor <andrew@sanpeople.com>
Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-at91.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/host/ohci-at91.c
+++ gregkh-2.6/drivers/usb/host/ohci-at91.c
@@ -187,7 +187,6 @@ ohci_at91_start (struct usb_hcd *hcd)
 {
 	struct at91_usbh_data	*board = hcd->self.controller->platform_data;
 	struct ohci_hcd		*ohci = hcd_to_ohci (hcd);
-	struct usb_device	*root = hcd->self.root_hub;
 	int			ret;
 
 	if ((ret = ohci_init(ohci)) < 0)
